servicerouter.js

```diff
@@ -205,6 +207,8 @@ class ServiceRouter {
                 resolve();
               })
               .catch((err) => {
+                this.appLogger.fatal(err.message);
+                this.appLogger.fatal(err.stack);
```

Comment:
This should probably be `this.appLogger.fatal(err)` (see https://github.com/pinojs/pino#pinostdserializerserr)